finalPrep/modZeroHero.js

```diff
@@ -48,24 +77,81 @@ var excuse = 'I think I forgot to lock up my 1992 Toyota Coralla. Be right back.
 //assessSituation(21, announcement, excuse) > should print - 'Never fear, the Courageous Curly Bracket is here!'
 //assessSituation(3, announcement, excuse) > should print - "Meh. Hard pass."
 
+function assessSituation2 (dangerLevel, announcement, excuse) {
+  if (dangerLevel >= 50){
+    console.log(excuse);
+  }
+  else if (dangerLevel < 10){
+    console.log('Meh. Hard pass.');
+  }
+  else {
+    console.log(announcement);
+  }
+}
+assessSituation2 (99, `${announcement}`, `${excuse}`);
```

Comment:
You can write this as `assessSituation2 (99, announcement, excuse);` - since you're not combining those variables with any text, you don't need interpolation!